### PR TITLE
feat: recording details interface output case path

### PR DESCRIPTION
**Pull request by pangdayuan1**

### CODE REVIEW

File: arex-storage-web-api/src/main/java/com/arextest/storage/repository/RepositoryProviderFactory.java [hunk 1]
```diff
@@ -56,6 +56,10 @@ public MockCategoryType findCategory(String categoryName) {
       return MockCategoryType.COVERAGE;
     }
 
+    if (MockCategoryType.RECORDING_SCENE.getName().equals(categoryName)) {
+      return MockCategoryType.RECORDING_SCENE;
```
Comment (lucas-myx) [inline arex-storage-web-api/src/main/java/com/arextest/storage/repository/RepositoryProviderFactory.java:60]: why not get from `categoryTypeMap.get(categoryName);` ?

Comment (pangdayuan1) [reply arex-storage-web-api/src/main/java/com/arextest/storage/repository/RepositoryProviderFactory.java:60]: There is only the default category type in categoryTypeMap

File: arex-storage-web-api/src/main/java/com/arextest/storage/service/ScheduleReplayingService.java [hunk 2]
```diff
@@ -110,11 +119,20 @@ public List<AREXMocker> queryRecordList(ViewRecordRequestType request) {
...
         }
       }
     }
 
+    // find recording scene data
+    if (findRecordingScene) {
+      AREXMocker arexMocker = recordingSceneMockerHandler.findByRecordId(recordId);
+      if (arexMocker != null) {
+        result.add(arexMocker);
+      }
```
Comment (lucas-myx) [inline arex-storage-web-api/src/main/java/com/arextest/storage/service/ScheduleReplayingService.java:133]: this if logic can it be merged onto `if (findRecordingScene)` ?

Comment (pangdayuan1) [reply arex-storage-web-api/src/main/java/com/arextest/storage/service/ScheduleReplayingService.java]: optimized

**Review by QizhengMo — APPROVED** [review]

File: arex-storage-web-api/src/main/java/com/arextest/storage/service/ScheduleReplayingService.java [hunk 3]
```diff
@@ -93,26 +95,41 @@ public List<AREXMocker> queryEntryPointByRange(PagedRequestType requestType) {
     return Collections.emptyList();
   }
 
-  public List<AREXMocker> queryRecordList(ViewRecordRequestType request) {
+  public List<AREXMocker> queryRecordList(ViewRecordRequestType request, SourceProvider sourceProvider) {
```
Comment (mr3) [inline arex-storage-web-api/src/main/java/com/arextest/storage/service/ScheduleReplayingService.java:98]: https://refactoringguru.cn/remove-assignments-to-parameters

File: arex-storage-web-api/src/main/java/com/arextest/storage/service/handler/mocker/MockerHandler.java [hunk 4]
```diff
@@ -1,5 +1,6 @@
 package com.arextest.storage.service.handler.mocker;
 
+import com.arextest.model.mock.AREXMocker;
```
Comment (mr3) [inline arex-storage-web-api/src/main/java/com/arextest/storage/service/handler/mocker/MockerHandler.java:3]: Remove useless import

File: arex-storage-web-api/src/main/java/com/arextest/storage/service/ScheduleReplayingService.java (hunk 3 above)
Comment (pangdayuan1) [reply arex-storage-web-api/src/main/java/com/arextest/storage/service/ScheduleReplayingService.java]: optimized, Use viewrecorddto as response model transmission

**Review by mr3 — APPROVED** [review]

**Review by sljoyy — APPROVED** [review]

**Pull request merged**